### PR TITLE
Generate and Send PDF

### PR DESCRIPTION
**Pull request by gigerbytes**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
+import os
 from flask import Flask, request, jsonify, render_template, send_file, Response
-from connect_database import connect_mongo, get_lifemap
-from twilio_helpers import twilio_send_template, send_messages
+from connect_database import connect_mongo, get_lifemap, save_number
+from twilio_helpers import send_template, send_messages, send_pdf
 import pdfkit
 
+BASE_URL = os.getenv("BASE_URL")
+PDF_DIR="./static/pdf/"
```
Comment (ahcai) [inline app.py:8]: nit: spaces between the `=`

File: connect_database.py [hunk 2]
```diff
@@ -8,19 +8,23 @@ def connect_mongo():
...
-        if 'lifeMap' in family: 
+        if 'lifeMap' in family:
             print(family['lifeMap'])
             # ** send_lifemap(family['lifeMap']) #this function has to be the one that sends the pdf to the WA
         else:
             # ** pdf=generatepdf(family)
-            # ** saves the pdf generated file in blob to save it in the db 
-            familyCol.update_one({'phoneNumber': phone_number},{'$set': {'lifeMap' : 'pdf'}}) 
+            # ** saves the pdf generated file in blob to save it in the db
+            familyCol.update_one({'phoneNumber': phone_number},{'$set': {'lifeMap' : 'pdf'}})
```
Comment (ahcai) [inline connect_database.py]: nit: space after comma

File: app.py [hunk 3]
```diff
@@ -20,10 +23,9 @@ def message_form():
 
 @app.route("/send-initial-message", methods=["POST"])
 def send_inital_message():
```
Comment (ahcai) [inline app.py:25]: i don't think this route is needed anymore?

File: connect_database.py [hunk 4]
```diff
@@ -8,19 +8,23 @@ def connect_mongo():
...
 
 def get_lifemap(phone_number):
     db = connect_mongo()
     familyCol = db['family']
     family = familyCol.find_one({'phoneNumber': phone_number}) # .Family needs to be the name of the collection where the familys are loaded
     if family :
         print(family)
-        if 'lifeMap' in family: 
+        if 'lifeMap' in family:
             print(family['lifeMap'])
```
Comment (yannieyip) [inline connect_database.py]: Is this `lifeMap` not `lifemap`? Just to double check

File: connect_database.py [hunk 5]
```diff
@@ -8,19 +8,23 @@ def connect_mongo():
...
 
+def save_number(phone_number):
+    db = connect_mongo()
+    numbers = db["numbers"]
+    numbers.insert({"number":phone_number})
 
 def get_lifemap(phone_number):
     db = connect_mongo()
     familyCol = db['family']
     family = familyCol.find_one({'phoneNumber': phone_number}) # .Family needs to be the name of the collection where the familys are loaded
```
Comment (yannieyip) [inline connect_database.py]: `families` not `familys` :)

File: connect_database.py (hunk 4 above)
Comment (gigerbytes) [reply connect_database.py]: changed to lifemapPdf

File: app.py [hunk 6]
```diff
@@ -32,20 +34,13 @@ def send_reminder():
...
-    numbers.insert({"number":phone_number})
-
-    twilio_send_template(phone_number)
-    return "lifemap sent!"
-
+    phone_number = request.form['From']
+    save_number(phone_number)
+    # pdfnetor(phone_number)
+    lifemap = send_pdf(phone_number)
+    return jsonify("pdf successfully sent")
```
Comment (yannieyip) [inline app.py:43]: If privacy isn't a concern, might be helpful to include the phone number in the return statement (e.g. "pdf successfully sent to " + phone_number)

Comment (gigerbytes) [reply app.py:43]: Would be a good idea to log this somewhere for future debugging/data purposes

**Pull request merged**